plugins/obs-outputs/ftl-stream.c

```diff
@@ -887,7 +888,11 @@ static void *status_thread(void *data)
 	while (!disconnected(stream)) {
 		if ((status_code = ftl_ingest_get_status(&stream->ftl_handle, &status, INFINITE)) < 0) {
 			blog(LOG_INFO, "ftl_ingest_get_status returned %d\n", status_code);
+#ifdef _WIN32
 			Sleep(500);
+#else
+			usleep(500000);
```

Comment:
while in the context of obs there is an os_sleep_ms() function that can be used instead of the #ifdefs